lib/board.rb

```diff
@@ -0,0 +1,44 @@
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
```

Comment:
You can just call `reset!` here since you are already initializing a new array there.